cocos/2d/CCAction.h

```diff
@@ -323,8 +323,15 @@ class CC_DLL Follow : public Action
      * @param followedNode  The node to be followed.
      * @param rect  The boundary. If \p rect is equal to Rect::ZERO, it'll work
      *              with no boundary.
-     */
-    static Follow* create(Node *followedNode, const Rect& rect = Rect::ZERO);
+     * @param xOffset The horizontal offset from the center of the screen from which the
+     *               node  is to be followed.It can be positive,negative or zero.If
+     *               set to zero the node will be horizontally centered followed.
+     *  @param yOffset The vertical offset from the center of the screen from which the
+     *                 node is to be followed.It can be positive,negative or zero.
+     *                 If set to zero the node will be vertically centered followed.
+     *   If both xOffset and yOffset are set to zero,then the node will be horizontally and vertically centered followed.
+     */
+    static Follow* create(Node *followedNode, const Rect& rect = Rect::ZERO,float xOffset=0.0,float yOffset=0.0);
     /** Return boundarySet.
```

Comment:
It is better to add a new `create` action according [cocos2d-x coding style](https://github.com/cocos2d/cocos2d-x/blob/v3/docs/CODING_STYLE.md#function-overloading).